extensions/data-plane/data-plane-spi/src/test/java/org/eclipse/dataspaceconnector/dataplane/spi/pipeline/ParallelSinkTest.java

```diff
@@ -41,6 +41,9 @@ class ParallelSinkTest {
     String dataSourceName = faker.lorem().word();
     String dataSourceContent = faker.lorem().characters();
     String errorMessage = faker.lorem().sentence();
+    InputStreamDataSource dataSource = new InputStreamDataSource(
```

Comment:
There are two tests that instantiate the same object as this, please use this everywhere or local instances everywhere (I'm more for the latter, but no strong opinion on it)